core/src/main/scala/algebra/PartialOrder.scala

```diff
@@ -158,6 +158,16 @@ object PartialOrder extends PartialOrderFunctions {
     }
 
   /**
+   * Define a `PartialOrder[A]` using the given functions `fs` until
+   * the first one returns a non-0 result or NaN.
+   */
+  def fromAll[@sp A](fs: ((A, A) => Double)*): PartialOrder[A] =
```

Comment:
maybe `fromFirst` here? Because as soon as you hit a difference you choose it, right? Not really all? Same comment above.